src/common/state/ColorTableAttributes.code

```diff
@@ -24,6 +24,23 @@ Postfix:
         return PyString_FromString(defaultDiscrete.c_str());
     }
 #endif
+#if VISIT_OBSOLETE_AT_VERSION(3,6,0)
```

Comment:
Quick question: was 'taggingFlag' ever saved to config files or session files?
If yes, then you need to add it to the  'ProcessOldVersions' function in this .code file.

Reply:
It should be there, at the bottom.

Reply:
My bad, thank you.